deploy-manage/security/set-up-basic-security.md

```diff
@@ -28,6 +28,14 @@ In this guide, you will learn how to:
 
 Refer to [Transport TLS/SSL settings](elasticsearch://reference/elasticsearch/configuration-reference/security-settings.md#transport-tls-ssl-settings) for the complete list of available settings in {{es}}.
 
+## Considerations for using an external CA
+
+You might choose to use an external CA to generate transport certificates for node-to-node connections.
```

Comment:
The word "external" suggests to me a third-party CA, but we also care about users using a CA that is private to their organization (but widely used within the organization) which they might consider to be "internal". Could we find some wording that covers both of these cases? Really it's any CA other than the one you can manage using the `elasticsearch-certutil` tool.